tensorboard/webapp/tbdev_upload/tbdev_upload_button_component.ts

```diff
@@ -33,6 +33,12 @@ const LOCAL_HOSTNAMES: string[] = ['localhost', '127.0.0.1'];
   `,
   styles: [
     `
+      /* More precise selector than angular material selectors to override
+         styling. */
+      button.mat-stroked-button {
```

Comment:
I implemented this pretty naively. Is there a more angular-friendly way of accomplishing this?

These are the selectors I'm competing with:
![image](https://user-images.githubusercontent.com/17152369/86829593-95c0f400-c062-11ea-980d-f917cd4f8482.png)